test/connectivity/NetworkController/NetworkInterfaceOfflineController.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System.Net.NetworkInformation;
+    using System.Runtime.InteropServices;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class NetworkInterfaceOfflineController : IOfflineController
```

Comment:
should it be named as "NetworkInterfaceController" since it has SetOffline and SetOnline methods?

Reply:
renamed to IController

---
In reply to: [353556306](https://github.com/Azure/iotedge/pull/2040#discussion_r353556306) [](ancestors = 353556306)